sequel-bulk-audit.gemspec

```diff
@@ -26,11 +26,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "sequel", ">= 4.0.0"
-  spec.add_dependency "pg", ">= 0.17.0"
+  spec.add_dependency "pg",     ">= 0.17.0"
 
-  spec.add_development_dependency "bundler", "~> 1.14"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency "pry", "~> 0.10"
-  spec.add_development_dependency "sequel_polymorphic"
+  spec.add_development_dependency "bundler", "~> 2.0"
+  spec.add_development_dependency "rake",    "~> 10.0"
+  spec.add_development_dependency "rspec",   "~> 3.0"
+  spec.add_development_dependency "pry",     "~> 0.10"
```

Comment:
may we also add 'rubocop-config-umbrellio' as a dev dependency? and update .rubocop.yml to use it

Reply:
Done.